app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="colorPrimary">#3F51B5</color>
+    <color name="colorPrimaryLight">#6272cc</color>
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
+    <color name="colorLine">#dcd4d0d0</color>
```

Comment:
Đặt theo format : color_polo_blue
Tham khảo ở đây : http://chir.ag/projects/name-that-color/#7D99CA